catalogue.json

```diff
@@ -114,6 +114,15 @@
                 "license": "Free",
                 "url": "https://cros.ec.europa.eu/dashboard/sdmx-converter",
                 "image": "https://ec.europa.eu/eurostat/o/adaptive-media/image/17995989/Thumbnail-128x128/EN_04_SDMX_Converter_API.png?t=1701270530811"
+            },
+            {
+                "name": "RJSDMX",
```

Comment:
Typo? shouldn't this be rsdmx?

Reply:
yes it's rsdmx, native R package, not the RJSDMX